source/routing/next_stop_time.cpp

```diff
@@ -413,42 +413,62 @@ static void fill_cache(const DateTime from,
     }
 }
 
-void CachedNextStopTime::load(const type::Data& data,
-        const DateTime from,
-        const DateTime to,
-        const type::RTLevel rt_level,
-        const type::AccessibiliteParams& accessibilite_params) {
+bool CachedNextStopTimeKey::operator<(const CachedNextStopTimeKey& other) const {
+    if (from != other.from) {
+        return from < other.from;
+    } else if (rt_level != other.rt_level) {
+        return rt_level < other.rt_level;
+    }
+    return accessibilite_params < other.accessibilite_params;
+}
+
+CachedNextStopTime CachedNextStopTimeManager::Fun::operator()(const CachedNextStopTimeKey& key) const {
+    CachedNextStopTime result;
+    auto logger = log4cplus::Logger::getInstance("log");
+    LOG4CPLUS_INFO(logger, "Cache miss : " << ++cache_miss);
```

Comment:
nop a fan of the instance variable incremented in the log.

If we change the log level we might change the value in the variable :wink: